src/core/lib/channel/server_config_selector.h

```diff
@@ -0,0 +1,39 @@
+//
```

Comment:
src/core/lib/channel is supposed to be generic channel related stuff that doesn't care about client or server.
Suggest a different location for this file.
Also suggest placing it somewhere in ext/? It seems like gRPC ought to be able to function without this.